### DISCUSSION TITLE
Add a link to /docs somewhere

### DISCUSSION THREAD

**Discussion by Spixmaster** [btcpayserver/btcpayserver#2347] [Ideas / Feature Requests]
**Describe the bug**
Could the endpoint `/docs` be linked somewhere like on the home page or the bar at the top. I have not seen it anywhere. The advantage would be that developers could look up the API documentation for their specific version on their own BTCPayServer instance.
I just found out about the endpoint `/docs` by reading docs.btcpayserver.org. That information was quite hidden. Are there still any useful endpoints I am not aware of? I think it would be a waste of data if such an important source of information like the API documentation would not be linked anywhere.

**Your BTCPayServer environment**
- BTCPay Server Version: 1.0.6.7
- Deployment Method: AUR package `btcpayserver-git`
- Browser: Brave

**Comment by pavlenex** [comment]
There's an effort to make it more prominent on the main website in [#139](https://github.com/btcpayserver/btcpayserver.org/issues/139) as far as the server interface itself goes, the front page will probably be obsoleted in the next few months with #1711 so adding it there would be a wasted efforts.
